doc/Development_Documentation/03_Documents/02_Document_Types/15_Print_Documents.md

```diff
@@ -95,6 +95,8 @@ Additionally, two settings can be used to add header and footer by the content o
 * header: URL for header content
 * footer: URL for footer content
 
+> Note: Only package spiritix/php-chrome-html2pdf:1.6.x is supported, not higher version due to dependency on Node > 18.
+
```

Comment:
Why not? We are currently using Node 18 just fine for Chrome Headless

Reply:
yes it works but then pimcore bundles like portal engine or statistics explorer are not supported on Node 18. There could be more side effects.

Reply:
Seems like a a potential security issue to revert back to abandoned versions of NPM though, no? If those specific bundles require older versions, why not specify in those composer files that they have specific requirements?

Reply:
Agree - closed, it's not a `pimcore/pimcore` issue. Thanks for your feedback.